tempo/kfserving/protocol.py

```diff
@@ -35,6 +35,16 @@ def create_v2_from_np(arr: np.ndarray, name: str) -> Dict:
         else:
             raise ValueError(f"Unknown numpy type {arr.dtype}")
 
+    @staticmethod
+    def create_v2_from_any(data: Any, name: str) -> Dict:
+        b = list(bytes(data, 'utf-8'))
```

Comment:
Does it have to be converted to bytes or would it be ok to just return as an arbitrary dictionary? Is this to ensure it is consistent with grpc proto schema?

Reply:
It needs to be bytes I think as its raw data